Lib/fontTools/ufoLib/glifLib.py

```diff
@@ -1191,6 +1191,10 @@ def _readGlyphFromTreeFormat1(
             haveSeenAdvance = True
             _readAdvance(glyphObject, element)
         elif element.tag == "unicode":
+            if element.get("hex") is None:
```

Comment:
While we're nitpicking:

suggest to take `v = element.get("hex")` out of the try/except and place it here, then test for `if v is None: ...`. This avoids calling `.get()` twice, and now that we've checked it, the second `.get()` should otherwise become `element["hex"]`.

Reply:
i've done that.